src/crafting.cpp

```diff
@@ -378,35 +378,26 @@ bool recipe::check_eligible_containers_for_crafting(int batch) const
     all.insert(all.end(), res.begin(), res.end());
     all.insert(all.end(), bps.begin(), bps.end());
 
-    for(item &prod : all) {
+    for(item & prod : all) {
         if (prod.made_of(LIQUID)) {
```

Comment:
You could flip this to the opposite condition and `continue` to reduce the indentation level a bit.